vismne/viz.py

```diff
@@ -1,36 +1,77 @@
 from .visuals import BrainMesh
 
 
-lh_viewdict = {'lateral': {'v': (180., 90.), 'r': 90.},
-               'medial': {'v': (0., 90.), 'r': -90.},
-               'rostral': {'v': (90., 90.), 'r': -180.},
-               'caudal': {'v': (270., 90.), 'r': 0.},
-               'dorsal': {'v': (180., 0.), 'r': 90.},
-               'ventral': {'v': (180., 180.), 'r': 90.},
-               'frontal': {'v': (120., 80.), 'r': 106.739},
-               'parietal': {'v': (-120., 60.), 'r': 49.106}}
-rh_viewdict = {'lateral': {'v': (180., -90.), 'r': -90.},
-               'medial': {'v': (0., -90.), 'r': 90.},
-               'rostral': {'v': (-90., -90.), 'r': 180.},
-               'caudal': {'v': (90., -90.), 'r': 0.},
-               'dorsal': {'v': (180., 0.), 'r': 90.},
-               'ventral': {'v': (180., 180.), 'r': 90.},
-               'frontal': {'v': (60., 80.), 'r': -106.739},
-               'parietal': {'v': (-60., 60.), 'r': -49.106}}
+lh_viewdict = {'lateral': {'v': (-90., 0.), 'r': 90., 'xyz': [1, 2]},
+               'medial': {'v': (90., 0.), 'r': -90., 'xyz': [1, 2]},
+               'rostral': {'v': (180., 0.), 'r': -180., 'xyz': [0, 2]},
+               'caudal': {'v': (0., 0.), 'r': 0., 'xyz': [0, 2]},
+               'dorsal': {'v': (0., 90.), 'r': 90., 'xyz': [0, 1]},
+               'ventral': {'v': (0., -90.), 'r': 90., 'xyz': [0, 1]},
+               'frontal': {'v': (120., 80.), 'r': 106.739, 'xyz': [0, 1]},
+               'parietal': {'v': (-120., 60.), 'r': 49.106, 'xyz': [0, 1]}}
+rh_viewdict = {'lateral': {'v': (90., 0.), 'r': -90., 'xyz': [1, 2]},
+               'medial': {'v': (-90., 0.), 'r': 90., 'xyz': [1, 2]},
+               'rostral': {'v': (180., 0.), 'r': 180., 'xyz': [0, 2]},
+               'caudal': {'v': (0., 0.), 'r': 0., 'xyz': [0, 2]},
+               'dorsal': {'v': (0., 90.), 'r': 90., 'xyz': [0, 1]},
+               'ventral': {'v': (0., -90.), 'r': 90., 'xyz': [0, 1]},
+               'frontal': {'v': (60., 80.), 'r': -106.739, 'xyz': [0, 1]},
+               'parietal': {'v': (-60., 60.), 'r': -49.106, 'xyz': [0, 1]}}
 viewdicts = dict(lh=lh_viewdict, rh=rh_viewdict)
 
 
 class Brain(object):
-    """docstring for Brain."""
-
-    def __init__(self, subject_id, hemi, surf, title=None, views=['lat'],
-                 offset=True):
+    """Class for visualizing a brain using multiple views in vispy.
+
+    Parameters
+    ----------
+    subject_id : str
+        subject name in Freesurfer subjects dir
+    hemi : str
+        hemisphere id (ie 'lh', 'rh', 'both', or 'split'). In the case
+        of 'both', both hemispheres are shown in the same window.
+        In the case of 'split' hemispheres are displayed side-by-side
+        in different viewing panes.
+    surf : str
+        freesurfer surface mesh name (ie 'white', 'inflated', etc.)
+    title : str
+        title for the window
+    views : list | str
+        views to use (lateral, medial, rostral, caudal, dorsal, ventral,
```

Comment:
can you use the same as in pysurfer, so I can easily test. That is: ['lat', 'med', 'fos',  'cau', 'dor' 'ven', 'fro', 'par'] ?